src/platform.cpp

```diff
@@ -229,7 +229,7 @@ static void imgui_end_frame() {}
 int main(int argc, char **argv) { // Game entrypoint
 #define ARGUMENT(LONG, SHORT) (std::strcmp((LONG), argv[index]) == 0 || std::strcmp((SHORT), argv[index]) == 0)
     int width = 500;
-    int height = 1000;
+    int height = 500;
```

Comment:
```suggestion
    int height = 1000;

Reply:
I see now that you named this commit "set the window dimensions to something sensible" but I don't think 500x1000 is a sensible resolution :)

Reply:
What about 720p? 1280x720.

Reply:
It's just horribly tall... I don't have a tall display...

Reply:
Sorry, I got the numbers backwards (oops). I thought this PR had set 500x1000, not revert it back to 500x500. I still think we could to 720p though.

Reply:
Actually, let's just stick with 500x500 for now.